async-kafka/apps/emergency-location-service/src/test/java/com/redhat/training/emergency/EmergencyLocationRouteBuilderTest.java

```diff
@@ -0,0 +1,88 @@
+package com.redhat.training.emergency;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.ConsumerTemplate;
+import org.apache.camel.builder.AdviceWithRouteBuilder;
+import org.apache.camel.model.RouteDefinition;
+import org.apache.camel.test.spring.CamelSpringBootRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@RunWith(CamelSpringBootRunner.class)
+@SpringBootTest
+public class EmergencyLocationRouteBuilderTest {
```

Comment:
I think that tests should use Mocks, to me is more important testing the logic we want.. how the data flows from an endpoint to another, than testing that the db/kafka components do their job writing data.

Reply:
I think a bit different in this case sorry. If we were using lab grading would we use mocks?

I tried to explain my thinking in this comment a bit: https://github.com/RedHatTraining/AD221/pull/238#discussion_r798021527